File: app/src/main/java/com/shuyu/apprecycler/chat/data/factory/ILocalChatDetailGetListener.java
package com.shuyu.apprecycler.chat.data.factory;

import com.shuyu.apprecycler.chat.data.model.ChatBaseModel;

import java.util.List;

/**
 * 获取本地聊天数据毁掉
 * Created by guoshuyu on 2017/9/11.
 */

public interface ILocalChatDetailGetListener {
    void getData(List<ChatBaseModel> datList);
}

File: app-kotlin/src/main/java/example/shuyu/recycler/kotlin/chat/data/factory/ILocalChatDetailGetListener.kt
package example.shuyu.recycler.kotlin.chat.data.factory


import example.shuyu.recycler.kotlin.chat.data.model.ChatBaseModel

/**
 * 获取本地聊天数据毁掉
 * Created by guoshuyu on 2017/9/11.
 */

open interface ILocalChatDetailGetListener {
    fun getData(datList: List<ChatBaseModel>)
}
